docs/geedocs/5.2.1/answer/7160001.html

```diff
@@ -33,7 +33,18 @@
 <p><img src="../art/common/googlelogo_color_260x88dp.png" width="130" height="44" alt="Google logo" /></p>
 <h1><a href="../index.html">Google Earth Enterprise Documentation Home</a> | Release notes</h1>
 <h2>Release notes: Open GEE 5.2.1</h2>
-<p>Open GEE 5.2.1 is actively under development</p>
+<p>Open GEE 5.2.1 is an incremental release of <a href="../answer/7160000.html">Open GEE 5.2.0</a>. It contains new features, several bug fixes in Fusion and GEE Server, and updates to third-party libraries.</p>
+  <h5>
+    <p id="overview_5_2_1">New Features</p>
+  </h5> 
+    <p><strong>MrSID support</strong>. Open GEE 5.2.1 build checks for MrSID libraries and if found, Fusion will be built with MrSID raster format support.</p>
```

Comment:
Instead of "Open GEE 5.2.1 build checks" maybe "At build time open GEE 5.2.1 now checks"

Reply:
sure